src/Adapter/MSTestAdapter.PlatformServices/Utilities/DeploymentItemUtility.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Uti
 /// </summary>
 internal class DeploymentItemUtility
 {
+    // REVIEW: it would be better if this was a ReflectionHelper, because helper is able to cache. But we don't have reflection helper here, because this is platform services dll.
```

Comment:
Can't we move the `ReflectionHelper` class here? The adapter references the platform services project so it would be available to both.

Reply:
maybe, it looked like a big change. Different issue and PR?

Reply:
I was also thinking about writer only lock, but I am not sure if concurrent dictionary does not do that for me automatically already.

Reply:
https://github.com/microsoft/testfx/issues/3043